Cargo.toml

```diff
@@ -7,7 +7,7 @@ members = ["egui_plot", "demo", "examples/*"]
 edition = "2021"
 license = "MIT OR Apache-2.0"
 rust-version = "1.76"
-version = "0.29.0"
```

Comment:
This is not just a dependency update; see https://github.com/emilk/egui_plot/blob/main/RELEASES.md#versioning

Reply:
Ok, I changed it to not touch egui crates ad the crate version and instead did that here: https://github.com/emilk/egui_plot/pull/54

I intended to get `egui_plot` to 0.29.1 like `egui` is. If these kinds of PRs are not welcome, I apologize for wasting your time.